tests/test1021.py

```diff
@@ -0,0 +1,19 @@
+#Test deleting points from contour
+
+import fontforge
+
+cont = fontforge.contour()
+cont.moveTo(0, 0).lineTo(10, 0).lineTo(20, 0).lineTo(30, 0)
+
+c = cont.dup()
+del c[1]
+assert len(c) == 3
+assert c[0].x == 0
+assert c[1].x == 20
+assert c[2].x == 30
+
+c = cont.dup()
+del c[1:3]
+assert len(c) == 2
```

Comment:
Something looks strange for me here. `del c[1:3]` is supposed to delete the elements 1, 2. But the remaining contour has length 2 and not 1. Could you please check?